src/sentry/static/sentry/app/views/eventsV2/eventQueryParams.tsx

```diff
@@ -4,6 +4,7 @@ export type ColumnValueType =
   | 'number'
   | 'duration'
   | 'timestamp'
+  | 'boolean'
```

Comment:
@leedongwei I had to add `boolean` here once I've provided the type hint for the `FIELDS` constant that's defined later in this file.